### PR TITLE
AdMob Integration

### LINKED ISSUE #45: AdMob integration
Integrate Admob in the app, it is currently in beta.
This issue can be assigned after the products screen is created.

### PR DESCRIPTION
**Pull request by sssash18**
### Related Issue
- Fixes #45 

### Proposed Changes
- Added Admob Integration

### Additional Information
- Real App ID need to be added
### Checklist
- [x] Tested
- [x] No Conflicts
- [x] Change In Code
- [ ] Change In Documentation

### Screenshots
Before this PR            | After this PR
:-----------------------: | :-----------------------:
** Original Screenshot ** | ** Updated Screenshot **

### CODE REVIEW

**Comment by sssash18** [comment]
@himanshusharma89  Please have a look.
Also change the app ID in Manifest as app is crashing due to this!

File: android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -24,6 +24,9 @@
               android:name="io.flutter.embedding.android.NormalTheme"
               android:resource="@style/NormalTheme"
               />
+            <meta-data
```
Comment (himanshusharma89) [inline android/app/src/main/AndroidManifest.xml:27]: This <meta-data> needs to be outside the <activity> but inside the <application> tag. Like here: https://github.com/googleads/googleads-mobile-flutter/blob/master/packages/google_mobile_ads/example/android/app/src/main/AndroidManifest.xml

**Comment by himanshusharma89** [comment]
Also @sssash18, you need to squash the commits, you don't need to commit every file with an update. Always commit all the updated files all at once at the end. Do the above change and squash the commit: https://github.com/wprig/wprig/wiki/How-to-squash-commits

**Review by himanshusharma89 — CHANGES_REQUESTED** [review]

**Comment by sssash18** [comment]
@himanshusharma89 Please review the changes and tell whether I am able to squash correctly ?

**Review by himanshusharma89 — CHANGES_REQUESTED** [review]
The other(not viewed) files need to be removed, these are your desktop generated files.

**Comment by himanshusharma89** [comment]
Also @sssash18, commits are not squashed. Try it again, or you can try with a particular commit and remove the previous ones before that one.

**Comment by sssash18** [comment]
@himanshusharma89  Please review I combined all changes in one commit.

**Pull request merged**